### DISCUSSION TITLE
How to upload paper against Rubric?

### DISCUSSION THREAD

**Discussion by akif-patel** [microsoft/ML-For-Beginners#206] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/13865031/126119254-f659fa1b-a008-4f0d-b40f-92274f72231a.png)

**Comment by jlooper** [comment, marked as answer]
Hi, this is a self study course. Those rubrics are supplied for your own reference or if a teacher wants to use this. However if you want feedback, copy a PAT for a lesson group from the list below, link your work, and ask for feedback here
